CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Changed
+
+- Migrate filelog operators to follow opentelemetry-log-collection v0.29.0 changes ([Migrate filelog operators to follow v0.29.0 changes #436](https://github.com/signalfx/splunk-otel-collector-chart/pull/436))
```

Comment:
nit to avoid repeat

```suggestion
- Migrate filelog operators to follow opentelemetry-log-collection v0.29.0 changes ([#436](https://github.com/signalfx/splunk-otel-collector-chart/pull/436))
```